lib/src/ui/movie_detail.dart

```diff
@@ -36,22 +47,21 @@ class MovieDetail extends StatelessWidget {
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
                     MovieDetailTitle(
-                      movieTitle: result.title,
+                      movieTitle: title,
                     ),
                     SizedBox(
                       height: MovieDimensions.sizedBoxHeight,
                     ),
                     MovieDetailSubtitle(
-                      movieReleaseDate: result.releaseDate,
-                      movieOriginalLanguage:
-                          result.originalLanguage.toUpperCase(),
-                      movieVoteCount: result.voteCount.toString(),
+                      movieReleaseDate: releaseDate,
+                      movieOriginalLanguage: originalLanguage.toUpperCase(),
+                      movieVoteCount: voteCount.toString(),
```

Comment:
If you use this as String, avoid to convert inside the widget, change voteCount to String, the same to originalLanguage, pass it converted to uppercase

Reply:
Done